apps/linode-marketplace-code-server/roles/code-server/defaults/main.yml

```
@@ -0,0 +1,3 @@
+---
+# Download path
+code_download_link: https://github.com/coder/code-server/releases/download/v{{ code_server_version }}/code-server_{{ code_server_version }}_amd64.deb
```

```suggestion
code_server_download_link: https://github.com/coder/code-server/releases/download/v{{ code_server_version }}/code-server_{{ code_server_version }}_amd64.deb
```

Just so that we can keep the semantics the same